### DISCUSSION TITLE
Known device, but doesn't work

### DISCUSSION THREAD

**Discussion by nomad-cyanide** [zigpy/zigpy#1385] [General]
Problem with newly purchased  Aqara Door & Windows Sensor T1. 
I have bought 2 more of these. I already have  22 of them installed and working around the house, but the 2 new ones won't work.

They pair just fine, with ZHA, but there are no sensors or battery-status, like all the other ones have and the new ones don't respond or react to anything. (like open/close)

If I look at the zigbee info for the new ones, they are called: lumi.magnet.agl02
Where the old ones are called:  lumi.sensor_magnet.aq2

Any suggestions to how I can get these to work as intended?

**Comment by Hedda** [comment]
Posted reply in HA forum -> https://community.home-assistant.io/t/non-functional-new-version-of-aqara-door-windows-sensor-t1/718519/
